cryptography/src/bls12381/primitives/poly.rs

```diff
@@ -40,21 +42,23 @@ pub struct Eval<C: Element> {
 
 impl<C: Element> Write for Eval<C> {
     fn write(&self, buf: &mut impl BufMut) {
-        self.index.write(buf);
+        UInt(self.index).write(buf);
```

Comment:
@BrendanChou this was what I meant by `PartialSignature` (but the change you made to `Share` was also worthwhile).

Reply:
Weird I thought I changed that one, must have missed it